main/src/ca/uwaterloo/flix/language/phase/Documentor.scala

```diff
@@ -47,7 +47,7 @@ object Documentor {
     //
     // Determine whether to generate documentation.
     //
-    if (!flix.options.documentor) {
+    if (!flix.options.documentorJson) {
       return root
```

Comment:
I think we should just return Unit here (see my earlier comments on the phases).